src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,148 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Scanner;
+
+/**
+ *
+ */
+public class Ui {
+    private Scanner sc;
+
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Prints welcome message during startup.
+     */
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        showLine();
+        printStandard("Hello! I'm Duke");
+        printStandard("What can I do for you?");
+        showLine();
+    }
+
+    /**
+     * Prints farewell message when "bye" is entered.
+     */
+    public void showExit() {
+        printStandard("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     *
+     */
+    public void showLoadingError() {
```

Comment:
what about add some information inside, like "duke.txt doesn't seem to be there or is incorrect"